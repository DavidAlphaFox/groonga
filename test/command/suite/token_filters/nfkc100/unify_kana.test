tokenize TokenDelimit "リンゴ りんご 林檎" \
  --token_filters 'TokenFilterNFKC100("unify_kana", true)'
