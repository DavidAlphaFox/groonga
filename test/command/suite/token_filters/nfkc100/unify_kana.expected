tokenize TokenDelimit "リンゴ りんご 林檎"   --token_filters 'TokenFilterNFKC100("unify_kana", true)'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "りんご",
      "position": 0,
      "force_prefix": false
    },
    {
      "value": "りんご",
      "position": 1,
      "force_prefix": false
    },
    {
      "value": "林檎",
      "position": 2,
      "force_prefix": false
    }
  ]
]
